src/components/molecules/ChatMessageBox/ChatMessageBox.tsx

```diff
@@ -47,7 +50,7 @@ export const ChatMessageBox: React.FC<ChatMessageBoxProps> = ({
 
   const sendMessageToChat = handleSubmit(({ message }) => {
     setMessageSending(true);
-    sendMessage(message);
+    sendMessage({ text: message, isQuestion });
```

Comment:
Once the change in `src/types/chat.ts` suggested in this review is made to `SendMessageProps`, this can be simplified to:
```suggestion
    sendMessage({ message, isQuestion });
```